diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 034999175e30316e488404aa5392f38792d8bdde..bd0dc150005f6976cf7ca51d153a70fd4086a822 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -323,6 +323,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
@@ -594,3 +595,5 @@ set_defaults("component") {
     configs = default_compiler_configs
   }
 }
+
+import("//brave/build/config/brave_build.gni")
