diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index b47217f4df0d587d7f3e5631d6c6772451a4f237..03b6f4dc2c921aa5320fe30d71540a613720e4ef 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -7725,6 +7725,7 @@ const FeatureEntry kFeatureEntries[] = {
     // "LoginCustomFlags" in tools/metrics/histograms/enums.xml. See "Flag
     // Histograms" in tools/metrics/histograms/README.md (run the
     // AboutFlagsHistogramTest unit test to verify this process).
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 };

 class FlagsStateSingleton : public flags_ui::FlagsState::Delegate {
