diff --git a/components/sync/engine/BUILD.gn b/components/sync/engine/BUILD.gn
index d9228903019a57c929427125641f558889662be5..99e5fc6400c0e1d60e949b86dde65913d23401b7 100644
--- a/components/sync/engine/BUILD.gn
+++ b/components/sync/engine/BUILD.gn
@@ -186,4 +186,5 @@ static_library("engine") {
     "//crypto",
     "//third_party/zlib/google:compression_utils",
   ]
+  sources += brave_components_sync_engine_sources
 }
